src/js/components/Tip/Tip.js

```diff
@@ -0,0 +1,50 @@
+import React, { forwardRef, useContext, useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+
+import { defaultProps } from '../../default-props';
+import { Box } from '../Box';
+import { Drop } from '../Drop';
+
+export const Tip = forwardRef(
+  ({ children, content, dropProps, ...rest }, tipRef) => {
+    const theme = useContext(ThemeContext) || defaultProps.theme;
+    const [over, setOver] = useState(false);
+    const ref = useRef();
+
+    return (
+      // Not sure about the extra Box, I'm still thinking of ways to simplify
+      <Box ref={tipRef} {...rest}>
+        <Box
```

Comment:
I think we should try to have less Boxes. I think we can do it with zero Boxes. Some ideas:
- use `cloneElement()` to add the event handlers and ref, rather than wrapping with a Box.
- return `[{clonedChild}, <Drop>]` when we render the Drop.